docs/Materials.md.html

```diff
@@ -2409,8 +2409,8 @@
 **getWorldGeometricNormalVector()**     | float3   |  Normalized normal in world space, before bump mapping (can be used before `prepareMaterial()`)
 **getWorldReflectedVector()**           | float3   |  Reflection of the view vector about the normal (must be used after `prepareMaterial()`)
 **getNormalizedViewportCoord()**        | float3   |  Normalized user viewport position (i.e. NDC coordinates normalized to [0, 1] for the position, [1, 0] for the depth), can be used before `prepareMaterial()`). Because the user viewport is smaller than the actual physical viewport, these coordinates can be negative or superior to 1 in the non-visible area of the physical viewport.
-**getMaterialOrientationMatrix()**      | float3x3 |  Rotation matrix to apply to bi/triplanar mapping. ![](images/shapr-logo.png)
 **getMaterialOrientationCenter()**      | float3   |  World space position to apply bi/triplanar mapping from. World space caveats do not apply. ![](images/shapr-logo.png)
+**getMaterialOrientation()**            | float4    |  Rotation represented as quaternion to apply to bi/triplanar mapping. ![](images/shapr-logo.png)  
```

Comment:
nit: I'd place this above `getMaterialOrientationCenter()`

Reply:
https://github.com/shapr3d/filament/pull/162/commits/9a6832a43a9510e842cf514172dd40d76045f21d